pkg/fleet/installer/setup/djm/emr.go

```diff
@@ -98,6 +111,13 @@ func SetupEmr(s *common.Setup) error {
 	if isMaster {
 		setupResourceManager(s, clusterName)
 	}
+	// Add logs config to both Resource Manager and Workers
+	sparkIntegration := s.Config.IntegrationConfigs["spark.d/conf.yaml"]
+	if os.Getenv("DD_EMR_LOGS_ENABLED") == "true" {
+		s.Config.DatadogYAML.LogsEnabled = true
+		sparkIntegration.Logs = emrLogs
+	}
+	s.Config.IntegrationConfigs["spark.d/conf.yaml"] = sparkIntegration
```

Comment:
could this be written in this way instead ?

```suggestion
	if os.Getenv("DD_EMR_LOGS_ENABLED") == "true" {
		s.Config.DatadogYAML.LogsEnabled = true
		s.Config.IntegrationConfigs["spark.d/conf.yaml"] = emrLogs
	}
```